packages/hs-react-ui/src/components/Button/Button.tsx

```diff
@@ -79,8 +79,8 @@ export const ButtonContainer: string & StyledComponentBase<any, {}, ButtonContai
     const shadowOpacity = 0.5 - elevation * 0.075;
 
     return `
-      display: inline-block;
-      font-size: 1em;
+      display: inline-flex;
+      font-size: 1rem;
```

Comment:
Kinda 50/50 on this one - but buttons should probably fit into the text around them
```suggestion
      font-size: 1em;
```